### DISCUSSION TITLE
Is  Free Online Editor open source?

### DISCUSSION THREAD

**Discussion by cdemircioglu** [gorules/zen#60] [Q&A]
Hi, 
I am really impressed with the Zen Engine. I would like to use it in one of my projects. However, I couldn't find the source code of "Free Online Editor", to create JDM json files. Is it available? 

Cheers,
Cem

**Comment by ivanmiletic** [comment]
Hi, that version isn't, but we are working on open source version of our JDM Editor React component:

https://github.com/gorules/jdm-editor

It is still in Draft but In the first version we are releasing Decision Table, and it will be followed by Decision Graph.
